pipeline/load.py

```diff
@@ -0,0 +1,220 @@
+"""This file is responsible for loading data into the RDS"""
+# pylint: disable=W0718, W0621
+
+import os
+import logging
+
+from psycopg2.extensions import connection, cursor
+from psycopg2 import OperationalError
+from dotenv import load_dotenv
+import psycopg2.extras
+
+from extract import extract_process
+from transform import transform_process
```

Comment:
move these into the `if __name__ == '__main__'` block

Reply:
That won't be possible since I've created a load_process function that uses functions from these two modules

Reply:
shouldnt they be passed in as a parameter instead? so that we can combine them in one single main.py file easily